src/libultra/io/controller.c

```diff
@@ -17,11 +17,11 @@ s32 osContInit(OSMesgQueue* mq, u8* bitpattern, OSContStatus* data) {
     OSTimer mytimer;
     OSMesgQueue timerMesgQueue;
 
-    if (D_80097E40 != 0) {
+    if (__osContinitialized != 0) {
```

Comment:
```suggestion
    if (__osContinitialized) {
```
This seems boolean? Could this also be `__osContInitialized` with the I capitalized?